LayoutKitTests/TextExtension.swift

```diff
@@ -44,12 +44,30 @@ extension Text {
                 let font = fontName.flatMap({ (fontName) -> UIFont? in
                     return UIFont(name: fontName, size: CGFloat(fontSize))
                 })
+
                 for text in texts {
-                    tests.append(TestCase(text: text, font: font))
+                    if let font = font {
+                        tests.append(TestCase(
+                            text: text.createFontAttribute(with: font),
```

Comment:
The behavior of `UITextView` is slightly different with `UILabel`.

For this case, if `UILabel().font` is called, you will get the default font of `UILabel`, but for `UITextView`, it doesn't. `UITextView().font` will get nil font and
```
let textView = UITextView()
textView.text = "" // default font is still unavailable
textView.text = " " // default font is avaiable
```
Also,
for the AppleTV, default font is different when you set text and attribute text
```
let textView = UITextView()
textView.text = " "
textView.attributeText = ... 
```

Let me try to tweak it inside of the test to see they can pass.